eclass/wine.eclass

```diff
@@ -359,6 +370,10 @@ wine_src_install() {
 	    dosym -r /usr/lib/fex-xtajit/libwow64fex.dll \
 				${WINE_PREFIX}/wine/aarch64-windows/xtajit.dll
 
+	use arm64ec &&
+	    dosym -r /usr/lib/fex-xtajit/libarm64ecfex.dll \
+				${WINE_PREFIX}/wine/aarch64-windows/xtajit64.dll
```

Comment:
I randomly spotted that the old xtajit.dll line above had weird spacing (tab + 4 spaces, then a tab too many) and fixed it in the tree but figured it'd have been copy/pasted for this one too:
```suggestion
		dosym -r /usr/lib/fex-xtajit/libarm64ecfex.dll \
			${WINE_PREFIX}/wine/aarch64-windows/xtajit64.dll
```